workflow/executor/executor.go

```diff
@@ -627,13 +627,19 @@ func (we *WorkflowExecutor) SaveLogs(ctx context.Context) []wfv1.Artifact {
 // saveContainerLogs saves a single container's log into a file
 func (we *WorkflowExecutor) saveContainerLogs(ctx context.Context, tempLogsDir, containerName string) (*wfv1.Artifact, error) {
 	fileName := containerName + ".log"
+	artName := containerName + "-logs"
+	if containerName == common.MainContainerName && we.Template.GetType() == wfv1.TemplateTypeContainer &&
+		len(we.Template.Container.Name) > 0 {
+		fileName = we.Template.Container.Name + ".log"
+		artName = we.Template.Container.Name + "-logs"
+	}
```

Comment:
Refactor these into constants?

Reply:
done, refactor `.log` and `-logs` into constants